fhir-bucket/src/main/java/com/ibm/fhir/bucket/app/Main.java

```diff
@@ -199,9 +200,19 @@ public class Main {
     // How many payer scenario requests do we want to make at a time.
     private int concurrentPayerRequests = 0;
 
+    // How many $bench requests to we want to make at a time.
```

Comment:
only if making other changes
```suggestion
    // How many $bench requests do we want to make at a time.
```

Reply:
fixed